build.gradle.kts

```diff
@@ -134,6 +134,9 @@ tasks.runIde {
     jvmArgs("-Djdk.module.illegalAccess.silent=true")
     jvmArgs("-XX:+UseCompressedOops")
 
+    jvmArgs("-Xms512m")
+    jvmArgs("-Xmx2048m")
```

Comment:
Does it need to be so high? One of the reasons for keeping the memory limit low is to make it easier to notice scenarios where IDE Perf uses too much memory. I think JetBrains does the same thing when developing intellij-community (they use `-Xmx512m` in their default run config).

Did you run out of memory when opening a very large project? Or was IDE Perf itself using a lot of memory?

Also note that you can create local run configurations in the IDE in order to use different JVM flags.

Reply:
```
    jvmArgs("-XX:ReservedCodeCacheSize=512m")
```
It looks like this parameter generates warnings when you start an IDE; Other params aren't too important.